### PR TITLE
[docs] Add status header for honeycomb exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for honeycomb exporter.

### CODE REVIEW

**Comment by dashpole** [comment]
Should this be deprecated instead of beta given the note at the top of the exporter?

**Review by codeboten — COMMENTED** [review]
Pinging codeowners @paulosman @lizthegrey @MikeGoldsmith, please take a look at @dashpole's question

**Comment by Frapschen** [comment]
hi,  @paulosman @lizthegrey @MikeGoldsmith can you have time to review this PR?

**Comment by MikeGoldsmith** [comment]
Yes, sorry - I was tracking this and it got lost on my list. 

I'll review today 👍🏻

**Comment by MikeGoldsmith** [comment]
Yes, I agree with @dashpole. We would prefer this to be marked as deprecated.

Honeycomb supports directly receiving OTLP over gRPC & HTTP/proto and that's our preferred ingest method.

**Comment by Frapschen** [comment]
@MikeGoldsmith mark honeycomb exporter  as deprecated.

File: exporter/honeycombexporter/go.mod [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// Deprecated: elasticexporter exporter is deprecated and will be removed in future versions.
```
Comment (MikeGoldsmith) [inline exporter/honeycombexporter/go.mod:1]:
```suggestion
// Deprecated: honeycombexporter exporter is deprecated and will be removed in future versions.
```

**Review by MikeGoldsmith — APPROVED** [review]
Looks good, thanks @Frapschen. Just left one minor update.

**Review by MikeGoldsmith — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Review by codeboten — COMMENTED** [review]
Please run the following to update `go.sum`:

`make -C exporter/honeycombexporter tidy`

**Pull request merged**